src/models/transfer.py

```diff
@@ -215,6 +238,29 @@ def from_slippage(cls, slippage: SolverSlippage) -> Transfer:
         """
         return cls(
             token=None,
-            receiver=slippage.solver_address,
+            solver=slippage.solver_address,
             amount_wei=slippage.amount_wei,
         )
+
+    @staticmethod
+    def sort_list(transfer_list: list[Transfer]) -> None:
+        """
+        This is the preferred and tested sort order we use for generating the transfer file.
+        It ensures that transfers are grouped
+        1. first by initial recipient (i.e. solvers),
+        2. then by token address (with native ETH last)
+        and finally order by amount descending (so that largest in category occur first).
+        Note that this method mutates the input data and nothing in returned.
+        """
+        transfer_list.sort(key=lambda t: (t.solver, str(t.token), -t.amount))
+
+    def __eq__(self, other: object) -> bool:
```

Comment:
Why is this method needed if we use a custom sort key?

Reply:
This was introduced for the purpose of unit tests (i.e. comparing transfers). You are correct to notice that it is not technically needed in the main program. What I could do to mitigate this is to implement a `TestTransfer` class that extends `Transfer` with this `__eq__` method in the test suite, but I think this is overkill (especially given that this Eq method is a valid representation of Transfer equality.